dotnet/src/SemanticKernel/SkillDefinition/InlineFunctionsDefinitionExtension.cs

```diff
@@ -43,7 +43,7 @@ public static class InlineFunctionsDefinitionExtension
         string? functionName = null,
         string? skillName = null,
         string? description = null,
-        int maxTokens = 256,
+        int? maxTokens = null,
```

Comment:
I think we should leave defined defaults here and above but still allow users to explicitly set them to null. Thoughts?

Reply:
My main reasoning for defaulting the base methods to null was because, in my opinion, anyone using an LLM that requires an explicit token count passed is already counting tokens and looking to set this value. If not, the API should handle setting a sensible default. Passing no value is more "future proof" as things evolve.

If an explicit value is to be kept, why is 256 (or any other count for that matter) a good default? Both OpenAI and Microsoft OpenAI completion API's default to 16. I'd argue this behavior is already strange in that the alignment there doesn't match up. We're also looking at tokens through how OpenAI tokenizes, what if a model releases where 256 is past the model limits or even not enough tokens to account for a full word? The developer should be fully aware of what LLM they are using and know whether or not they should be passing a token limit or letting the API default to what it needs.